projects/plugins/jetpack/_inc/client/components/plans/plan-icon/index.jsx

```diff
@@ -108,7 +112,10 @@ const PRODUCT_ICON_MAP = {
 	[ PLAN_JETPACK_COMPLETE_MONTHLY ]: '/plans/jetpack-complete.svg',
 	[ PLAN_JETPACK_VIDEOPRESS ]: '/products/product-jetpack-videopress.svg',
 	[ PLAN_JETPACK_VIDEOPRESS_MONTHLY ]: '/products/product-jetpack-videopress.svg',
-
+	[ PLAN_JETPACK_SOCIAL_BASIC ]: '/plans/jetpack-complete.svg',
```

Comment:
Do we need Social icons here? We should be able to use a variation on the SVG icon we use elsewhere.